src/components/Base/Popover/index.scss

```diff
@@ -0,0 +1,115 @@
+@import '~scss/variables';
+
+.popover {
+  :global {
```

Comment:
`:global` should be in top-level block

Reply:
It is worth while to discuss the problem. @leoendless

Reply:
``:global`` under ``.popover`` will only effect elements under local style ``.popover``

Reply:
got it